Makefile

```diff
@@ -39,13 +39,13 @@ osx-init:
 	brew install go@1.14
 	$(MAKE) envinit
 servewallet:
-	go run -mod=vendor ./cmd/servewallet
+	go run -mod=vendor ./cmd/servewallet/...
```

Comment:
I don't believe you need triple dot. Does it not work for you without it?

Reply:
Right, fixed